tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature

```diff
@@ -58,3 +58,37 @@ Feature: checking file versions using file id
     And using new DAV path
     When user "Brian" tries to get the number of versions of file "/text.txt" using file-id path "/meta/<<FILEID>>/v"
     Then the HTTP status code should be "403"
+
+
+  Scenario Outline: check the file versions of a file after moving to a shared folder inside a project space as editor/viewer
+    Given user "Alice" has created a folder "testFolder" in space "Project1"
+    And user "Alice" has created a share inside of space "Project1" with settings:
+      | path      | testFolder |
+      | shareWith | Brian      |
+      | role      | <role>     |
+    When user "Alice" moves file "text.txt" from space "Project1" to "/testFolder/movedText.txt" inside space "Project1" using the WebDAV API
+    Then the HTTP status code should be "201"
+    When using new DAV path
+    And user "Alice" gets the number of versions of file "/testFolder/movedText.txt" using file-id path "/meta/<<FILEID>>/v"
+    Then the HTTP status code should be "207"
+    And the number of versions should be "1"
+    When user "Brian" tries to get the number of versions of file "/Shares/testFolder/movedText.txt" using file-id path "/meta/<<FILEID>>/v"
+    Then the HTTP status code should be "403"
+    Examples:
+      | role   |
+      | editor |
+      | viewer |
+
+
+  Scenario: check the file versions of a file after moving to a shared folder inside a project space
```

Comment:
@fschade Does this scenario confirm this statement in https://github.com/owncloud/ocis/issues/7738
>rewrite the tests to use a project instead of a personal space, the NewSpaceEditorRole roleSet contains the ListFileVersions permission.

Sharee can see the file versions